src/view/com/util/forms/PostDropdownBtnMenuItems.tsx

```diff
@@ -396,7 +397,9 @@ let PostDropdownMenuItems = ({
     const video = post.embed as AppBskyEmbedVideo.View
     const did = post.author.did
     const cid = video.cid
-    const uri = `https://bsky.social/xrpc/com.atproto.sync.getBlob?did=${did}&cid=${cid}`
+    if (!did.startsWith('did:')) return
```

Comment:
note to myself: could probably lift this into a type and a type predicate, but don't need to do that here lol